tests/cli_test.py

```diff
@@ -90,7 +97,7 @@ def test_simple(runner):
 
     result = runner.invoke(main_khal)
     assert not result.exception
-    assert result.output == 'No events\n'
+    assert strip_ansi_escapes(result.output) == 'No events\n'
```

Comment:
Please do. Maybe there was something odd in my environment, but merely redefining `strip_ansi_escapes` to `return value` will show this issue, though *only on python 3!*